CHANGELOG.md

```diff
@@ -2742,7 +2742,7 @@ v3.16.0 - February 20, 2017
 * c7e64f3 Upgrade: mock-fs (#8070) (Toru Nagashima)
 * acc3301 Update: handle uncommon linebreaks consistently in rules (fixes #7949) (#8049) (Teddy Katz)
 * 591b74a Chore: enable operator-linebreak on ESLint codebase (#8064) (Teddy Katz)
-* 6445d2a Docs: Add documentation for /* exported */ (fixes #7998) (#8065) (Lee Yi Min)
+* 6445d2a Docs: Add documentation for /*exported*/ (fixes #7998) (#8065) (Lee Yi Min)
```

Comment:
The change log is auto generated, so we shouldn’t make edits. Can you revert these changes?